.github/workflows/test.yml

```diff
@@ -43,10 +43,17 @@ jobs:
             name: test with OCB
             makepot: "true"
           - container: ghcr.io/oca/oca-ci/py3.6-odoo14.0:latest
-            exclude: "product_tax_multicompany_default"
+            include: "stock_intercompany"
             name: test with Odoo
           - container: ghcr.io/oca/oca-ci/py3.6-ocb14.0:latest
-            exclude: "product_tax_multicompany_default"
+            include: "stock_intercompany"
+            name: test with OCB
+          - container: ghcr.io/oca/oca-ci/py3.6-odoo14.0:latest
+            exclude: "product_tax_multicompany_default,stock_intercompany"
+            makepot: "true"
```

Comment:
```suggestion
            makepot: "true"
          - container: ghcr.io/oca/oca-ci/py3.6-odoo14.0:latest
            exclude: "product_tax_multicompany_default,stock_intercompany"
```

Reply:
It actually might be the solution, since it was generating recursively lots of commits with `.pot` files.